### ISSUE TITLE
FYI Broken in MacOS Catalina

### ISSUE THREAD

**Issue by Rizora** [unixpickle/JamWiFi#30]
It should come as no surprise that this is broken in the beta... I myself haven't figured out the problem though

**Comment by ReduB3** [comment]
Me too :'(

**Comment by CharlieWhile13** [comment]
I'll work on trying to fix it :)

**Comment by xazsqa** [comment]
Mac OS Catalina broken...

**Comment by max1on1** [comment]
> I'll work on trying to fix it :)

Hi, is there any progress in solving the problem?

**Comment by wootwoot1234** [comment]
@CharlieWhile13 Were you able to fix the crashing issue?

**Comment by CharlieWhile13** [comment]
I’m still on it!

**Comment by irfanrosandi** [comment]
is there any update guys?

**Comment by elihwyma** [comment]
Yes in the read me of this, there is a new repo which is updated
